get-started/02_our_app.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Our Application"
+title: "Sample application"
```

Comment:
Food for thought; for the page title itself, wondering if we should include "get started" on all of these pages. Doing so may help providing more context to the page.

Could be;

```
Get started: Sample application
```

or (if we want to include "what step" we're in)

```
Get started, step 1: Sample application
```


Currently, it shows:

<img width="780" alt="Screenshot 2021-02-03 at 12 29 28" src="https://user-images.githubusercontent.com/1804568/106741371-de05bc00-661b-11eb-86b1-21877c73a2ec.png">




Alternatively, we could also look at modifying the template to add a suffix to the page-title (`<title>`) that differs from the H1, so that Google would show;

```
Sample application | Get started | Docker Documentation
```

or

```
Sample application | Docker Documentation: Get started
```

Reply:
Adding Get started on each page would be a repetition. Updating the suffix might be a better option. Will think about this. Let's discuss during our next sync.